model/src/pyrenew/latent/infections.py

```diff
@@ -49,14 +49,14 @@ class Infections(RandomVariable):
 
     def __init__(
         self,
-        infections_mean_varname: str = "latent_infections",
+        infections_varname: str = "latent_infections",
     ) -> None:
         """
         Default constructor for Infections class.
 
         Parameters
         ----------
-        infections_mean_varname : str, optional
+        infections_varname : str, optional
             Name to be assigned to the deterministic variable in the model.
             Defaults to "latent_infections".
```

Comment:
```suggestion
        infections_varname : str
            Name to be assigned to the deterministic variable in the model.
```

As per [DHM](https://github.com/CDCgov/multisignal-epi-inference/pull/205#pullrequestreview-2131069672)'s comment.